virttest/utils_test/libvirt.py

```diff
@@ -2576,9 +2576,16 @@ def get_all_vol_paths():
 
 def do_migration(vm_name, uri, extra, auth_pwd, auth_user="root",
                  options="--verbose", virsh_patterns=".*100\s%.*",
-                 su_user="", timeout=30):
+                 su_user="", timeout=30, return_ret_output=False):
     """
     Migrate VM to target host.
+
+    :param  return_ret_output: True to return both result and output,
+            otherwise return only result
+
+    :return: True if migration ok, otherwise False
+    :return: The output message of migration session
```

Comment:
2 returns contradict to each other?

since you updated the docstring, pls add all other params together.